packages/charts/src/charts/line/line-chart.component.tsx

```diff
@@ -28,6 +35,10 @@ export type Props = {
   minValue?: number | 'auto';
   /** Maximum value for axis */
   maxValue?: number | 'auto';
+  /** X axis title settings */
+  xAxisTitle?: AxisTitleType;
```

Comment:
The `AxisTitle` should be `string | number` primitive type. The position alignment should be a part of `Theme` object - that could be different for each axis.